__tests__/index.test.js

```diff
@@ -0,0 +1,38 @@
+const app = require("../app.js");
+const request = require("supertest");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const data = require("../db/data/test-data/index");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+
+describe('GET /api/topics', () => {
+    test('It should respond with an array of topics', async () => {
+        const response = await request(app).get('/api/topics');
+        expect(response.statusCode).toBe(200);
+        expect(response.body).toBeInstanceOf(Array);
+        expect(response.body[0]).toHaveProperty('slug');
```

Comment:
We need to check more than the first element in an array. We should iterate the array and assert again **each** of the elements. This can be done with an array method and asserting inside that. We want to make assertions  for each element 😉 

If you do use this array method we also MUST have an assertion _in the same test_ that the array is of the correct length. This protects against a false positive if there was an empty array or no array returned by the endpoint.